tests/steps/docling/test_docling_step.py

```diff
@@ -23,6 +31,8 @@
     ],
 )
 def test_docling_step(real_data_path, expected_md_start, expected_contract_count):
+    if IS_MACOS and os.environ.get("GITHUB_ACTIONS") == "true":
```

Comment:
```suggestion
    if IS_MACOS and os.environ.get("GITHUB_ACTIONS") == "true":
```
I'd love for this to be a default fixture in `tests/steps/docling/conftest.py`
So thath also all future pytests are skipped